bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/Enums.java

```diff
@@ -139,6 +138,15 @@ public enum HomekitMode {
         public String getValue() {
             return value;
         }
+
+        public static HomekitMode fromValue(String value) throws IllegalArgumentException {
+            for (HomekitMode m : HomekitMode.values()) {
+                if (m.getValue().equals(value)) {
+                    return m;
+                }
+            }
+            throw new IllegalArgumentException("Unexpected HomekitMode value of \"" + value + "\"");
+        }
```

Comment:
Why create this while java already offers `HomekitMode.valueOf(String)` ?

Reply:
good idea, I'll use Java's valueOf